Software/GuitarPedal/Effect-Modules/eq_module.cpp

```diff
@@ -0,0 +1,149 @@
+#include "eq_module.h"
+#include <q/fx/biquad.hpp>
+
+constexpr uint8_t NUM_FILTERS = 6;
+
+using namespace bkshepherd;
+
+const float minGain = -15.f;
+const float maxGain = 15.f;
+
+const float centerFrequency[NUM_FILTERS] = {100.f, 200.f, 400.f, 800.f, 1600.f, 3200.f};
+const float q[NUM_FILTERS] = {0.7f, 0.8f, 1.1f, 1.4f, 1.6f, 1.8f};
```

Comment:
chatgpt helped me with these and I compared it to my other multifx pedal and it each knob sounds VERY similar so I am happy with these!

Reply:
Also I felt like a fool because chatgpt basically typed out my entire implementation in a handful of seconds from 1 simple prompt https://chatgpt.com/share/67724377-ac34-8012-8ee5-4c6b15c32241 , wild stuff